### DISCUSSION TITLE
Header lines divide table cell and make from one cell two different cells

### DISCUSSION THREAD

**Discussion by gkukawski** [jsvine/pdfplumber#1093] [Ask for help with specific PDFs]
Hi. I have a problem with one pdf.
It's just a table and I want to read it as a table.
But there is some additional lines on the page. It looks like header od pdf.
It exists on each page.
It causes that firs row is divided and it makes two records in result instead of one.

How can I fix that?
THanks in advanced for your help.

![image](https://github.com/jsvine/pdfplumber/assets/719348/cefdd17d-5ea9-4638-970a-e9339f43d550)

**Comment by jsvine** [comment]
Hi @gkukawski, and thanks for sharing that `debug_tablefinder(...)` output. Without access to the PDF itself, it's difficult to provide detailed suggestions, but in general:

- Examine the objects in `page.rects` and `page.lines`. Do there seem to be attributes that distinguish the lines you want from those you don't?
- If so, you can use `filtered = page.filter(...)` to remove those (you can see some examples of this in other discussions), and then `filtered.extract_table(...)`.
